requirements/eval-deps.pip

```diff
@@ -21,3 +21,5 @@ sympy~=1.13
 typing-extensions~=4.12
 tzdata~=2024.2
 yarl~=1.13
+plotly~=5.24
+kaleido==0.1.*
```

Comment:
Would be nice to sort these

Reply:
78935236270f83de9b1692e91620bd259e6c313d